source/includes/documentation/_vue_js_elements.md

```diff
@@ -4991,3 +4991,150 @@ __Vuetify v2.6.14__ code example:
+                        <div class="v-calendar-daily__day-interval" style="height: 48px;"></div>
+                        <...>
+                        <div class="v-event-timed-container">
+                            <div class="v-event-timed primary white--text"
+                                 style="top: 600px; height: 144px; left: 0%; width: 100%;">
+                                <div class="pl-1">
+                                    <span class="v-event-summary"><strong>Mash Potatoes</strong><br>12:30 PM - 3:30 PM</span>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <...>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+```
+
+| Method | Description                                                            | Return Type |
+| :--- |:-----------------------------------------------------------------------| :--- |
+**activeDay()** | Get active date of Calendar                                            | int
+**isDailyType()** | Get if Calendar has daily type                                         | boolean
+**isWeeklyType()** | Get if Calendar has weekly type                                | boolean
+**hasCategories()** | Get if Calendar has categories                                 | boolean
+**hasDayIntervals()** | Get if Calendar has intervals                                  | boolean
+**getCategory(int catNum)** | Get Calendar's required category name                          | String
+**getDayInterval(int intNum)** | Get Calendar's required interval text                          | String
+**isToday()** | Check that Calendar has the current day                                | boolean
+**dailyEvent(int eventNum)** | Get Calendar required event summary          | UIElement
+**selectSlot(int week, int day, int slot)** | Select Calendar's slot             | void
+**slotTitle(int week, int day, int slot)** | Get slot's title          | String
+**hasCurrentTimeLine()** | Get if Calendar has current time line                                  | boolean
+**displayedDaysOfMonth()** | Returns a list of  days that are visible now                           | List<WebElement>
+**events()** | Returns a list of events that are visible on a Calendar                | WebList
+**intervals()** | Returns a list of intervals of a calendar                              | WebList
+**intervalHeaders()** | Returns a list of intervals headers of a calendar                      | WebList
+**intervalBody()** | Gets a parent interval element that contains all inetrvals of calendar | UIElement
+**dayEvents(int day)** | Returns a list of timed events from a required day                     | WebList
+**calendarDays()** | Returns a list of calendar's days                                      | WebList
+**eventRipple(int eventNumber)** | Gets a required ripple event       | UIElement
```

Comment:
according to [documentation](https://v2.vuetifyjs.com/en/api/v-calendar/#props) this method could be here. but it is not used in any test yet. In my opinion, it is because I can not see this attribute in any calendar's  implementation